### PR TITLE
Change how full grid refreshes from websockets work

### PR DESCRIPTION
**Pull request by siemiatj**
Related to #10276 . Impact is limited to grid views only and only in reaction to a websocket event, so it's safe to merge anytime.

https://recordit.co/zfdKI8KD3p

### CODE REVIEW

File: frontend/src/containers/DocumentList.js [hunk 1]
```diff
@@ -459,7 +471,7 @@ class DocumentListContainer extends Component {
    * @method getData
    * @summary Loads view/included tab data from REST endpoint
    */
-  getData = (id, page, sortingQuery, locationAreaSearch) => {
+  getData = (id, page, sortingQuery, locationAreaSearch, websocketRefresh) => {
```
Comment (petrican) [inline frontend/src/containers/DocumentList.js:474]: too many params. Can we use object param in here ?

File: frontend/src/actions/ViewActions.js [hunk 2]
```diff
@@ -416,7 +422,7 @@ export function fetchDocument({
         }
 
         // get quickactions
-        if (viewId && shouldFetchQuickActions) {
+        if (viewId && shouldFetchQuickActions && !websocketRefresh) {
```
Comment (petrican) [inline frontend/src/actions/ViewActions.js:425]: As I understand from Teo this is NOK as quickactions shall be refreshed

**Review by petrican — CHANGES_REQUESTED** [review]

**Comment by siemiatj** [comment]
Ok, so now we're also refreshing quick actions and not showing the spinner overlay at all https://recordit.co/nZ6Gl8yDX3

File: frontend/src/containers/DocumentList.js (hunk 1 above)
Comment (siemiatj) [reply frontend/src/containers/DocumentList.js:474]: Dude this is a friggin' hack I want to remove as soon as possible and not spend a second more on it than needed.

**Review by petrican — APPROVED** [review]
lgtm

**Pull request merged**